vm/src/vm/mod.rs

```diff
@@ -242,6 +242,7 @@ impl VirtualMachine {
         let mut essential_init = || -> PyResult {
             #[cfg(not(target_arch = "wasm32"))]
             import::import_builtin(self, "_signal")?;
+            import::import_builtin(self, "_ast")?;
```

Comment:
```suggestion
            #[cfg(any(feature = "parser", feature = "compiler"))]
            import::import_builtin(self, "_ast")?;
```
if it is only required by compile function, feature checking may be helpful.